scripts/fetch.sh

```diff
@@ -40,7 +40,7 @@ update_repo() {
 		git fetch origin || die "unable to fetch ${uri}"
 	else
 		echo "Cloning '${path}'"
-		if [ -z "${GIT_LOCAL_REF_DIR}" ]; then
+		if [ -z "${GIT_LOCAL_REF_DIR}" -a ! -d "${GIT_LOCAL_REF_DIR}" ]; then
```

Comment:
yeah I dropped the whole check cause it didn't really matter. But yeah there it needed to be an or.